src/instantsearch/widget.js

```diff
@@ -5,23 +5,23 @@ import places from '../places.js';
  */
 class AlgoliaPlacesWidget {
   constructor({ defaultPosition, ...placesOptions }) {
-    if (defaultPosition instanceof Array && defaultPosition.length === 2) {
+    if (Array.isArray(defaultPosition) && defaultPosition.length === 2) {
       this.defaultPosition = defaultPosition.join(',');
     }
 
     this.placesOptions = placesOptions;
     this.placesAutocomplete = places(this.placesOptions);
 
-    this.state = {};
+    this.stateSetDuringRouting = false;
+    this.query = '';
   }
 
   init({ helper }) {
-    helper
-      .setQueryParameter('insideBoundingBox')
-      .setQueryParameter(
-        'aroundLatLng',
-        this.state.position || this.defaultPosition
-      );
+    if (!this.stateSetDuringRouting) {
```

Comment:
Rather than set the position inside the `init` we could use the [`getConfiguration`](https://community.algolia.com/instantsearch.js/v2/guides/custom-widget.html#the-widget-lifecycle-and-api) function. This lifecycle is called before the `init` and the purpose of it is to set initial parameters. Thoughts @bobylito ?

Reply:
Good idea! It should even prevent the need for the boolean flag.

Reply:
Thanks for the info. `getConfiguration` is now implemented.